include/Managers/Managers.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef MANAGERS_HPP
+#define MANAGERS_HPP
+
+#include <Arduino.h>
+#include "Context.hpp"
+#include "Messages.hpp"
+#include "Drivers/MotorController.hpp"
+
+namespace manager {
+
+const size_t NUM_MOTORS = 2;
```

Comment:
Should this be part of the motor manager class? Don't think it should belong to the manager namespace in general

Reply:
You're right, made it a member of motor manager.